### DISCUSSION TITLE
Daily 29/11/2023

### DISCUSSION THREAD

**Discussion by Sinan-Karakaya** [Sinan-Karakaya/R-Type#22] [Daily]
29/11/2023

**Comment by Sinan-Karakaya** [comment]
Going to finish the Catch2 integration
Then I'll start working on the CPack integration
And hopefully I'll start working on the engine then

**Reply by Sinan-Karakaya** [reply]
Maybe i'll also work on a release CI with CPack

**Comment by lucasdpt** [comment]
Continue to write network protocol & start network lib on game engine

**Comment by jovanHillion** [comment]
je vais sans doute pas avoir le temps de taffer sur le r-type, mais je vais regarder un peu plus en détail le scripting
